examples/bind_imported_app_to_offcluster_operator_managed_AWS_RDS_db/README.md

```diff
@@ -0,0 +1,307 @@
+# Binding an Imported app to an Off-cluster Operator Managed AWS RDS Database
+
+## Introduction
+
+This scenario illustrates binding an imported application to an off-cluster operated managed AWS RDS Database. The scenario also shows how to bind the backing service to two different independent applications and a use of the `envVarPrefix` feature of the Service Binding Operator to specify prefix for the names of the injected environment variables.
```

Comment:
Small grammar nit - Should read "to specify a prefix"

Reply:
Thanks @ldimaggi... fixed.